ts/import-csv/HtmlSwitch.svelte

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import Col from "../components/Col.svelte";
+    import Row from "../components/Row.svelte";
+    import Switch from "../components/Switch.svelte";
+    import * as tr from "../lib/ftl";
+
+    export let isHtml: boolean;
+</script>
+
+<Row --cols={2}>
+    <Col --col-size={1}>
+        {tr.importingAllowHtmlInFields()}
+    </Col>
+    <Col --col-size={1} --col-justify="flex-end">
+        <Switch id={undefined} bind:value={isHtml} />
+    </Col>
+</Row>
```

Comment:
As we only have a single switch at the moment this is not a big deal, but in the long run we probably don't want to be creating separate files for each switch we add - it would be a bit nicer to have a reusable SwitchRow that allows passing in the boolean and text. We have a SwitchRow in deck-options that is similar, though we don't really need the revert button in this case. We could potentially make the revert button optional, or create a separate SwitchRow here. @hgiesel is the most qualified in this area, and he may have a preference on the way we should do it.

Reply:
Agreed, I'd generalise HtmlSwitch, if there was more than one use case, and ideally SwitchRow would be reused, but I was afraid to mess up existing pages. I'm not very skilled with the styling of different-level components.

Reply:
Sounds good. And me either :-)